### ISSUE TITLE
On Foundry v12, when I enable this module the entire tab for music/sounds disappears, for all users.

### ISSUE THREAD

**Issue by WylerL** [Matthew-Whitlock/FoundryVTT-music-permissions#8]
What it says on the tin it seems like this module doesn't function at all for v12. A shame, because I was really looking forward to using it in my upcoming game.

**Comment by Matthew-Whitlock** [comment]
Yeah, this probably isn't compatible with v12. But I believe you can do all of the same stuff with built-in features starting with v12! Take a look at the permission configuration for the different roles. 

But let me know if there are things missing from the core stuff. I can take a look at updating this if so

**Comment by WylerL** [comment]
> Yeah, this probably isn't compatible with v12. But I believe you can do all of the same stuff with built-in features starting with v12! Take a look at the permission configuration for the different roles.
> 
> But let me know if there are things missing from the core stuff. I can take a look at updating this if so

Ah awesome. I hadn't realized they'd added exactly what I was looking for. I think there needs to be a GM online for it to work, so I haven't gotten to test it yet but a player being able to control the audio is what I was hoping to accomplish, and supposedly that is a core feature now. Thanks!
